src/components/mote/components/innkalling/InnkallingDialogmotePanel.tsx

```diff
@@ -64,6 +86,7 @@ export const InnkallingDialogmotePanel = (): ReactElement => {
         header={texts.planleggNyttMote}
         subtitle={texts.ingenMoterPlanlagt}
       >
+        {!skalHaVarsel ? <BrukerKanIkkeVarslesWarning /> : <></>}
```

Comment:
Kan det funke med `!skalHaVarsel && <BrukerKanIkkeVarslesWarning />`, så slipper man ternary?

Reply:
Det fungerer hvis man endrer typen til children i `DialogmotePanel` til `ReactNode`. Kan godt gå for det.